### PR TITLE
[REVIEW] add google sheets service functions

### PR DESCRIPTION
**Pull request by rachitamaharjan**
## Description of the change

[Asana link](https://app.asana.com/0/0/1202310263555543/f)

### CODE REVIEW

File: data/config.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package data
```
Comment (Swechhya) [inline data/config.go:1]: Lets keep this inside googl_sheet folder

File: data/config.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+//GoogleCreds google service account credentials
+type GoogleCreds struct {
+	Type            string `json:"type" bson:"type"`
```
Comment (Swechhya) [inline data/config.go:5]: We don not need bson and json tags here.

**Review by Swechhya — CHANGES_REQUESTED** [review]

**Review by SudipKatwal — APPROVED** [review]

**Review by Swechhya — APPROVED** [review]

**Review by chawocecil — APPROVED** [review]

File: pkg/network/README.md [hunk 3]
```diff
@@ -1,3 +1,70 @@
...
+package main
+
+import (
+	"github.com/phil-inc/pcommon/pkg/network/google_sheets"
+)
+
+func main() {
+		var clearRanges []string
+		gc := google_sheets.GoogleCreds{
+      Type:            "type",
```
Comment (varanasiaditya) [inline pkg/network/README.md:38]: Formatting.

**Comment by Swechhya** [comment]
@rachitamaharjan  Can you please resolve the merge conflicts?

**Pull request merged**